modules/standard/BigInteger.chpl

```diff
@@ -3163,30 +3163,54 @@ When ``n/d`` does not produce an integer, this method may produce incorrect resu
   proc bigint.remove(const ref a: bigint, const ref f: bigint) : uint {
     return this.removeFactor(a,f);
   }
+  /*
+    .. warning::
 
+       a and f are deprecated - please use x and fac respectively
+  */
+  pragma "last resort"
+  deprecated
+  "a and f are deprecated - please use x and fac respectively"
```

Comment:
removeFactor is a new function for your PR, you don't need to have a deprecation warning when changing the argument names :)

Reply:
sure, I'll correct it.